src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private int defaultSize = 10;
+    private int maxSize = 50;
+    private int maxBound = 5;
```

Comment:
Avoid using magic numbers directly in the code. Define `defaultSize`, `maxSize`, `maxBound`, and any other magic numbers as constants to improve code readability and maintainability.